features/other_b/subscriptions.feature

```diff
@@ -172,6 +172,63 @@
   When I press "Subscribe"
   Then the page title should include "Chapter 2"
 
+  Scenario: Subscription to a work in an anonymous collection
```

Comment:
I think this test might be a bit more complicated than strictly necessary (a lot of the existing tests in here appear to be /o\). This might work:
```
Scenario: When a chapter is added to an anonymous work, subscription emails are sent
to users who have subscribed to the work

Given the anonymous collection "test_collection"
  And I am logged in as "first_user"
  And I post the work "Multi Chapter Work" to the collection "test_collection"
When I am logged in as "second_user"
  And I view the work "Multi Chapter Work"
  And I press "Subscribe"
When I am logged in as "first_user"
  And a chapter is added to "Multi Chapter Work"
  And subscription notifications are sent
Then 1 email should be delivered to "second_user@foo.com"
  And the email should contain "Multi Chapter Work"
  And the email should contain "Anonymous"
  And the email should not contain "first_user"
```

The other new tests might benefit from being similarly tweaked in places, including making the descriptions a bit more detailed.